name_echo.py

```diff
@@ -0,0 +1,3 @@
+print('What is your name?')
+name = input('-->')
+print('Hello ' + name +'.')
```

Comment:
Small note, there's a missing space between the final + operator and the next quotation.